src/js/grapl-cdk/lib/pipeline_dashboard.ts

```diff
@@ -0,0 +1,35 @@
+import * as cdk from '@aws-cdk/core';
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
+import { Service } from './service';
+
+function invocationsWidget(service: Service, retry?: boolean): cloudwatch.GraphWidget {
```

Comment:
would you update the parameter name `retry?` to something like `is_retry?`

it's hard to tell at a glance whether `retry?` means "this step IS a retry" vs. "this step will try and then retry"